sample-cap/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "type": "module",
+  "name": "@sap-ai-sdk/sample-cap",
+  "version": "1.0.0",
+  "description": "Sample CAP application with Cloud SDK for AI.",
+  "repository": "https://github.com/sap/ai-sdk-js",
+  "license": "UNLICENSED",
```

Comment:
[q] I guess this license is not intended. I propose to simply remove this line to avoid confusion.